app/src/main/res/drawable/doha_rating_colors.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+  <item>
+    <color android:color="#ffd677" />
```

Comment:
I have a suggesting for this to make our life easier when we define the theme in the apps codebase. Can we add instead of this three themeable props? E.g. `ratingLowColor`, `ratingNormalColor` and `ratingHighColor` and have one default `layer-list` that uses those variables. 

This will make it way easier to theme things, and we can still keep the api as it is (using the layer-list).

Reply:
Done